.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            2
+        ],
+        "quotes": [
+            1,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ]
+    },
+    "env": {
+        "node": true
+    },
+    "extends": "eslint:recommended"
```

Comment:
Leaving out rule "valid-jsdoc" as it introduces over 200 errors.
Added rules "no-multi-spaces", "no-multi-str", "no-negated-condition", and "padded-blocks (never)" as they introduce 0 errors.

Added rule "no-sync" but set it to warning only, it adds 2 warnings.